src/components/NavBar/NavBarPresentation.js

```diff
@@ -49,7 +49,7 @@ function NavBarPresentation({ isHome, isSearchPage, menuData, title }) {
               } md:flex items-center text-white`}
             >
               <span className="sr-only">FirstVoices Logo</span>
-              {fvlogo}
+              <Link to="/">{fvlogo}</Link>
```

Comment:
This looks great. Could you please have a peak at AppNavBarPresentation as well and make sure it's correct?